src/utils/walletIntegration/wallets/hoogiiWallet.tsx

```diff
@@ -81,8 +83,9 @@ class hoogiiWallet implements WalletIntegrationInterface {
     });
   }
 
-  disconnect(): void {
+  disconnect(): void | boolean {
```

Comment:
why not only `boolean`?

Reply:
Fixed https://github.com/Yakuhito/tibet-ui/pull/18/commits/8f7939446ae51d8d2db04e2935f1c367ef5e6c6f